packages/perl_read_key.rb

```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Perl_read_key < Package
+  description 'Dependency, fixes perl_read_key_pm'
+  homepage 'https://metacpan.org/source/JSTOWE/TermReadKey-2.37/'
+  version '2.37'
+  source_url 'https://github.com/jonathanstowe/TermReadKey/archive/v2.37.tar.gz'
+  source_sha256 '0fa4fb2f8145e3fb2c2129ad28d55be175abcc258f239ba8ddc2cd83790aa8fb'
+
+  def self.build
+    system "perl", "Makefile.PL"
+    system "make"
+  end
+
+  def self.install
+    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+  end
+
+  def self.postinstall
+    puts "Please install perl_read_key_pm after installing this package,"
+    puts "This just fills in dependencies so it can compile correctly."
+    puts "If you are currently installing perl_read_key_pm, you are OK."
```

Comment:
Would adding `depends_on 'perl_read_key'` to perl_read_key_pm.rb remove the need for this message?  Also, you are missing an `end`. :)  Please test to make sure these actually install before creating the PR.  Please make all your commits locally until you are ready.